src/services/user-config-service.ts

```diff
@@ -137,8 +137,8 @@ const convertYamlToUserConfig = (input?: string): Config => {
 	return output;
 };
 
-const updateRepoConfigFromGitHub = async (repoSyncState: RepoSyncState, githubInstallationId: InstallationId): Promise<void> => {
-	const yamlConfig = await getRepoConfigFromGitHub(githubInstallationId, repoSyncState.repoOwner, repoSyncState.repoName);
+const updateRepoConfigFromGitHub = async (repoSyncState: RepoSyncState, githubInstallationId: InstallationId, jiraHost: string, gitHubAppId?: number): Promise<void> => {
```

Comment:
Hey, just FYI I previously created a PR regarding the `?` for all these `gitHubAppId`. 
https://github.com/atlassian/github-for-jira/pull/1530

I think it should be fine here, as this is internal method, but only FYI, on external method I think using `gitHubAppId: number | undefined` is better.

Reply:
Thanks @gxueatlassian , changed that to `number | undefined`